src/index.js

```diff
@@ -0,0 +1,150 @@
+const state = {
+    temp: 70,
+    city: 'Seattle',
+};
+
+const tempInfo = document.getElementById('tempInfo');
+const tempUp = document.getElementById('tempUp');
+const tempDown = document.getElementById('tempDown');
+const landScape = document.getElementById("landScape");
+const cityInput = document.getElementById('cityInput');
+const cityValue = document.getElementById('cityInput').value;
+const currentCity = document.getElementById('currentCity');
+const realTemp = document.getElementById('getTemp');
+const skySelect = document.getElementById('skies');
+const sky = document.getElementById('sky').value;
+const resetCity = document.getElementById('resetCity')
+
+const updateWeather = () => {
+    tempInfo.textContent = state.temp
+};
+
+const increaseWeather = () => {
+    state.temp += 1;
+    updateWeather();
+    updatelandScape();
+};
+
+const decreaseWeather = () => {
+    state.temp -= 1;
+    updateWeather();
+    updatelandScape();
+};
+
+const updatecityInput = () => {
```

Comment:
Be sure to follow camelCase convention when naming methods, this should read `updateCityInput`